src/Product.php

```diff
@@ -626,6 +629,26 @@ public function processImages($data)
         return $this;
     }
 
+    /**
+     * @return array|null
+     */
+    protected function getAvailableImage(array $data)
+    {
+        foreach ($data as $image) {
+            if (empty($imageAvailabilities = $image['availabilities'])) {
```

Comment:
This variable should be assigned before the foreach, because otherwise it wouldn't be declared anyway.